graylog2-web-interface/src/logic/telemetry/TelemetryInit.test.tsx

```diff
@@ -37,11 +36,14 @@ jest.mock('util/AppConfig', () => ({
   telemetry: jest.fn(() => mockedTelemetryConfig),
 }));
 
-jest.mock('posthog-js', () => ({
-  ...posthogMock,
-  init: jest.fn(),
-  debug: jest.fn(),
-}));
+jest.mock('posthog-js', () => {
```

Comment:
Do we really need to mock the module here? It looks like we are just returning it unchanged.

Reply:
True, I forgot to clean it up after an iteration.